integration-tests/features/environment.py

```diff
@@ -76,9 +76,9 @@ def exec_file(filename, globals=None, locals=None):
     from _pytest import config
     from _pytest.assertion import rewrite
     f = path.local(filename)
-    filename2 = os.path.relpath(filename, os.getcwd())
     config = config._prepareconfig([], [])
-    _, code = rewrite._rewrite_test(config, f)
+    source_stat, code = rewrite._rewrite_test(config, f)
+    print('filename: {} source_stat: {} code: {}'.format(filename, source_stat, code))
```

Comment:
remove print

Reply:
I saw old code Print was used for adding logs, so followed the same. Its a test code, I don't think it will harm.

Reply:
Anyway **Prints** are only for debugging and should not slip into `master` in any way. Use Logging for any debbugging purposes.